src/pages/myNotes/myNotes.jsx

```diff
@@ -0,0 +1,32 @@
+import { Box } from '@material-ui/core';
+import React, { useState } from 'react';
+import { NOTES_LIST } from '../../assets/data/index';
+import Note from './note';
+import NoteText from './noteText';
+import useStyles from './styled';
+
+const MyNotes = () => {
+  const text = 'Select note to display';
+  const [defaultText, activeText] = useState({
```

Comment:
The second param from useState should started from `set..` words (`setActiveText`). It will help to make code clear.

Reply:
And maybe it will be `setActiveNote` instead?